crypto/include/cipher_types.h

```diff
@@ -74,6 +74,9 @@ extern srtp_debug_module_t srtp_mod_aes_icm;
 #ifdef OPENSSL
 extern srtp_debug_module_t srtp_mod_aes_gcm;
 #endif
+#ifdef MBEDTLS
```

Comment:
This is getting wordy (as opposed to just doing `#if defined(OPENSSL) || defined(MBEDTLS) || defined(NSS)`).  And don't we have a `GCM` variable anyway?  Seems like we should use `#ifdef GCM`, or if that doesn't work, the `#if defined || defined || defined` as above.

Reply:
If you make this change then great other wise we can do it in a separate clean up PR.

Reply:
I did not go through the nss part in detail, and see the nss in the cmakefiles either. So I am not sure it includes the gcm or not. Went through the nss part quickly, and it should have the gcm part. I changed it as the second choice #if (defined(OPENSSL) || defined(MBEDTLS) || defined(NSS)).